src/DynamicObj/ImmutableDynamicObj.fs

```diff
@@ -100,10 +100,11 @@ type ImmutableDynamicObj internal (map : Map<string, obj>) =
             | :? 'a as o -> o |> Some
             | _ -> None
 
-    // Merges two DynamicObj (Warning: In case of duplicate property names the members of the second object override those of the first)
-    static member combine (targetObject : #ImmutableDynamicObj) (sourceObject : #ImmutableDynamicObj) =
-
-        let rec loop (targetObject : #ImmutableDynamicObj) (sourceObject : #ImmutableDynamicObj) (propsToCopy: (string*obj) list) (acc:ImmutableDynamicObj) =
+    // Merges two ImmutableDynamicObj (Warning: In case of duplicate property names the members of the second object override those of the first)
+    static member combine (targetObject : 'a :> ImmutableDynamicObj) (sourceObject : 'b :> ImmutableDynamicObj) =
```

Comment:
I'd change this to a `///`-comment, with the note in a `remarks` element, so that this potentially-relevant detail is available to users:

```fsharp
/// <summary>Merges two ImmutableDynamicObj</summary>
/// <remarks>Warning: In case of duplicate property names the members of the second object override those of the first.</remarks>
```

Reply:
Thanks! Added

Reply:
If we the properties of the second object replaces those of the first object, it might also be more consistent to keep the second objects type instead of the first one

Reply:
which is also better for pipelines IMO as

```fsharp
'A
|> IDO.combine 'B
```
can be chained into functions that take type A as input:

```fsharp
'A
|> IDO.combine 'B
|> A.doSomething
```